src/pyFAI/ext/CSC_common.pxi

```diff
@@ -136,7 +137,7 @@ cdef class CscIntegrator(object):
         :param absorption: Apparent efficiency of a pixel due to parallax effect
         :type absorption: ndarray
         :param normalization_factor: divide the valid result by this value
-
+        :param bool weighted_average: set to False to use an unweigted mean (similar to legacy) instead of the weigted average WIP
```

Comment:
```suggestion
        :param bool weighted_average: set to False to use an unweighted mean (similar to legacy) instead of the weighted average WIP
```